Fixed Alignment Of Cards

Tejas2909 opened the pull request.

BhavyaSheth22 commented:
Thanks, just one change, can you please remove the learn more button from the testimonial cards? It doesn't do anything, and is not responsive.
![Screenshot from 2021-04-13 11-05-02](https://user-images.githubusercontent.com/60400357/114501825-5a360480-9c48-11eb-9f6d-846d1467d364.png)

Tejas2909 commented:
hello bhavya actually after alignment of cards i continued the same code files and  yesterday made another PR for order display so 
i actually dont know how to make changes to a previous PR so i updated the orders PR and removed the button in it . so you may merge this PR and once the Orders PR get merged the Learn more button would be removed . Thanks
